docs/rfcs/0XY-storcon-rollout.md

```diff
@@ -0,0 +1,106 @@
+# Feature Rollout on Storage Controller
+
+This RFC describes the rollout interface from a user's perspective. I do not have a concreate implementation idea
+yet, but the operations described below should be intuitively feasible to implement -- most of them map to a single
+SQL inside the storcon database and then some reconcile operations.
+
+The rollout RFC makes it possible for the storcon to gradually modify tenant configs based on filters and percentages.
+
+What will it look like if we want to rollout gc-compaction gradually?
+
+Create a feature called gc-compaction.
+
+```
+$ storcon-cli feature create --job gc-compaction
```

Comment:
Storing these in the storage controller would mean that we could have different sets of feature flags in different regions -- hard to keep track of compared with having a global store